src/nxdoc/nuxeo-server/installation.md

```diff
@@ -29,9 +29,16 @@ version_override:
   '6.0': 60/admindoc/installation
   '5.8': 58/admindoc/installation
 ---
+{{! excerpt}}
+Nuxeo Platform LTS 2021 can be deployed with the **Nuxeo Docker image** (recommended) or using the **ZIP distribution**. It can be installed on all major operating system such as Windows (client or server), Linux distributions, Mac OS, etc.
```

Comment:
The way we had presented it initially, putting the "Tomcat server ZIP" section as an additional INFO, was to really discourage people from using the ZIP. We had understood that Nuxeo clearly wanted to push towards Docker, instead of suggesting 2 alternatives at the same level. But maybe it isn't the case.

Reply:
Yet, if it makes sense like this, we might as well link `Nuxeo Docker image` and `ZIP distribution` to the related sections in the page.